vizro-core/examples/kpi/app.py

```diff
@@ -2,113 +2,128 @@
+        vm.Figure(
+            id="kpi-reverse-coloring",
+            figure=kpi_card_reference(
+                df_kpi_cards,
+                value_column="Total Complaints_2019",
+                reference_column="Total Complaints_2018",
+                title="Total Complaints",
+                value_format="{value:.0f}",
+                reference_format="{delta_relative:+.1%} vs. 2018 ({reference:.0f})",
+                icon="person",
+            ),
         ),
-        KPI(
-            title="Open Complaints",
-            value="0.4%",
-            icon="arrow_circle_down",
-            sign="delta-pos",
-            ref_value="-0.2% vs. LY",
+        vm.Figure(
+            figure=kpi_card_reference(
+                df_kpi_cards,
+                value_column="Closed Complaints_2019",
+                reference_column="Closed Complaints_2018",
+                title="Closed Complaints",
+                value_format="{value:.1f}%",
+                reference_format="{delta:+.1f}pp  vs. 2018 ({reference:.1f}%)",
+                icon="inventory",
+            )
         ),
-        KPI(
-            title="Timely Response",
-            value="98.1%",
-            icon="arrow_circle_up",
-            sign="delta-pos",
-            ref_value="+10.5% vs. LY",
+        vm.Figure(
+            figure=kpi_card_reference(
+                df_kpi_cards,
+                value_column="Timely response_2019",
+                reference_column="Timely response_2018",
+                title="Timely Response",
+                value_format="{value:.1f}%",
+                reference_format="{delta:+.1f}pp  vs. 2018 ({reference:.1f}%)",
+                icon="timer",
+            )
         ),
-        KPI(
-            title="Closed w/o cost",
-            value="84.5%",
-            icon="arrow_circle_down",
-            sign="delta-neg",
-            ref_value="-8.5% vs. LY",
+        vm.Figure(
+            figure=kpi_card_reference(
+                df_kpi_cards,
+                value_column="Closed w/o cost_2019",
+                reference_column="Closed w/o cost_2018",
+                title="Closed w/o cost",
+                value_format="{value:.1f}%",
+                reference_format="{delta:.1f}pp vs. 2018 ({reference:.1f}%)",
+                icon="payments",
+            )
         ),
-        KPI(
-            title="Consumer disputed",
-            value="9.5%",
-            icon="arrow_circle_up",
-            sign="delta-neg",
-            ref_value="+2.3% vs. LY",
+        vm.Figure(
+            figure=kpi_card_reference(
+                df_kpi_cards,
+                value_column="Consumer disputed_2019",
+                reference_column="Consumer disputed_2018",
+                title="Consumer disputed",
+                value_format="{value:.1f}%",
+                reference_format="{delta:+.1f}pp vs. 2018 ({reference:.1f}%)",
+                icon="sentiment_dissatisfied",
+            )
```

Comment:
Do you get a white square from the screenshot for the "Consumer disrupted" KPI card?

<img width="371" alt="image" src="https://github.com/user-attachments/assets/616f89a2-4140-441c-a1f0-0109e06f1722">

Reply:
Yes, unfortunately we can't get rid of this. It's the native scrolling bar corner 😬

Reply:
I succeeded to get rid of this with:
```py
kpi_banner = FlexContainer(
    id="kpi_banner_flex_container",
    components=[
    ...
)
```

and custom css:

```css
#kpi_banner_flex_container{
  overflow: hidden;
}
```

Reply:
Yes, but unfortunately it doesn't function properly anymore then.

The only reason why I put it into a FlexContainer is that this functions as a KPI banner, meaning if you reduce the width of the screen, the cards will take a minimum width but there will be scrolling horizontally. This ensures that the cards are always well visible and do not look too squeezed.

However, if you turn off the overflow, they will be just cut-off, and parts of the cards will not visible.

Reply:
I see it now! thanks for the explanation 😃